nightly-benchmark/publish_benchmark.py

```diff
@@ -62,8 +62,8 @@
   Profile](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-rand-access-scheduler.html)
 - [Simple
   Profile](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-simple-scheduler.html)
-- [Anon
-  Mean](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-anon-mean-scheduler.html)
+- [anom
```

Comment:
```suggestion
- [Anom
```

Reply:
Hehe James was faster! 🐎 😂  See ( https://github.com/quasiben/dask-scheduler-performance/pull/32#discussion_r532876821 ). Marking this as resolved so we can address up there 😉